exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,12 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr.length === 0) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
You could also make this function shorter, as it is not necessary to use if statement if the function is returning a boolean. Because result of comparison is a boolean itself. `return arr.length === 0` - returns a boolean.

Reply:
I agree, the shorter way is better as it is readable :) good effort!